Web/Github/git-repo-stars.1h.sh

```diff
@@ -0,0 +1,115 @@
+#!/usr/local/bin/node
+
+# <bitbar.title>Github Repo Stars</bitbar.title>
+# <bitbar.version>v1.0</bitbar.version>
+# <bitbar.author>Varun Malhotra</bitbar.author>
+# <bitbar.author.github>softvar</bitbar.author.github>
+# <bitbar.image>http://varunmalhotra.xyz/media/images/bitbar-plugin-github-repos-stars.png</bitbar.image>
+# <bitbar.desc>Github repos stars and Emoji notification 🔔 on change in stars count</bitbar.desc>
+# <bitbar.dependencies>node</bitbar.dependencies>
+# <bitbar.abouturl>http://varunmalhotra.xyz/blog/2016/02/bitbar-plugins-for-github-and-producthunt.html</bitbar.abouturl>
```

Comment:
`#` isn't a valid comment character in node. Which also made me notice that the extension for this file should be `.js` not `.sh`